source/development-tasks/deploy-a-branch-to-staging/index.html.md.erb

```diff
@@ -73,56 +67,53 @@ To put the everything back to how it was before:
 
 ## Deploy a branch from your machine
 
-If the branch you want to deploy is not on GitHub, you can deploy it
-with the CloudFoundry CLI.
+If the branch you want to deploy is on your machine instead of GitHub, you can deploy that branch with the CloudFoundry CLI.
 
-1. Log in to Concourse
+1. Sign in to Concourse:
 
     ```
     fly -t cd-govuk-tools login
     ```
 
-2. Disable deployments to staging
+2. Disable deployments to staging:
 
     ```
     fly -t cd-govuk-tools pause-job --job govuk-account-manager-prototype/deploy-app-staging
     ```
 
-3. Log in to CloudFoundry
+3. Log in to CloudFoundry:
 
     ```
     cf login -a api.london.cloud.service.gov.uk --sso -o govuk-accounts -s staging
     ```
 
-    If you are already logged in, select the staging space
-  
+    When you are logged in, select the staging space:
+
```

Comment:
@jonathanglassman This instruction is actually an "if", not a "when". Maybe it should be something like

3. Sign in to the staging environment:

    If you are already signed in to CloudFoundry:

    ```
    cf target -s staging
    ```

    If you are not already signed in to CloudFoundry:

    ```
    cf login -a api.london.cloud.service.gov.uk --sso -o govuk-accounts -s staging
    ```

Reply:
ah fair enough,